android/jni/com/mapswithme/maps/editor/Editor.cpp

```diff
@@ -246,8 +246,7 @@ Java_com_mapswithme_maps_editor_Editor_nativeSaveEditedFeature(JNIEnv *, jclass)
   case osm::Editor::NothingWasChanged:
   case osm::Editor::SavedSuccessfully:
     return true;
-  case osm::Editor::NoFreeSpaceError:
-    return false;
+  default: return false;
```

Comment:
Please, roll back this change. It's better to handle all cases in the switch statement instead of "default", because you'll get compile warnings in case of unexpected enum update.